nav_msgs/Odometry odom
# Point cloud messages
sensor_msgs/PointCloud2 filtered_pointcloud   # 滤波后的关键帧点云